triage/README.md

```diff
@@ -27,9 +26,11 @@ However, summarization can be run directly with the following flags:
 - `output_slices` (optional): a pattern to be used when outputting slices, if desired (see
   [Methodology](#methodology)); e.g. `slices/failure_data_PREFIX.json`, where `PREFIX` will be replaced
   with some identifier
-- `num_workers`: the number of worker goroutines to spawn for parallelized functions; defaults to `2*runtime.NumCPU()-1`. (Since CPU detection is unreliable in Kubernetes, we set it manually according to the number of CPUs in [test-infra-periodics.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/test-infra/test-infra-periodics.yaml).)
+- `num_workers` (optional): the number of worker goroutines to spawn for parallelized functions; defaults to `2*runtime.NumCPU()-1`. (Since CPU detection is unreliable in Kubernetes, we set it manually according to the number of CPUs in [test-infra-periodics.yaml](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/test-infra/test-infra-periodics.yaml).)
 - `memoize` (optional): whether to memoize certain function results to JSON (and use previously memoized results if they exist); defaults to false
 
+After all named flags are passed in, a space-delimited series of paths to files containing test information should be passed in as well.
```

Comment:
I might make this
```
- `...files` after all named flags are passed in, a space-delimited series of paths to files containing test information should be passed in as well.
```
Just so if people take a quick glance at the args list they don't miss this

Reply:
👍 done